shipitscript/task.py

```diff
@@ -3,23 +3,14 @@
 
 import scriptworker.client
 
-from shipitscript.exceptions import TaskVerificationError, BadInstanceConfigurationError
+from shipitscript.exceptions import TaskVerificationError
 from shipitscript.utils import get_single_item_from_sequence
 
 log = logging.getLogger(__name__)
 
-ALLOWED_API_ROOT_PER_VALID_SCOPE = {
-    'project:releng:ship-it:production': 'https://ship-it.mozilla.org',
-    'project:releng:ship-it:staging': 'https://ship-it-dev.allizom.org',
-    'project:releng:ship-it:dev': '*',
-}
 
-VALID_SCOPES = tuple(ALLOWED_API_ROOT_PER_VALID_SCOPE.keys())
-PROTECTED_API_ROOTS = tuple([
-    url
-    for url in ALLOWED_API_ROOT_PER_VALID_SCOPE.values()
-    if url != '*'
-])
+# TODO: Make this prefix a param of the instance config, when Thunderbird migrates this task
+_VALID_SCOPES_PREFIX = 'project:releng:ship-it:'
```

Comment:
Hm, I think this is already encapsulated in the instance config, but it can't hurt.
We have 2 scopesets per signing task, for example: one for cert level, and the other for signing format. If we ever add another behavior to this script, we might add a scope to specify which to use.

Reply:
Agreed.